include/flexflow/parallel_tensor.h

```diff
@@ -52,13 +56,28 @@ struct ParallelDim {
   bool is_replica_dim = false;
 };
 
+/**
```

Comment:
Are there any differences we should be aware of in terms of documenting non-header-files and how that interacts with doxygen?

Reply:
You can still document the non-header files and the document will show up on doxygen as well. But generally since header files define the API, people prefer to add more detailed documentation to the header files, especially the public API. If a public function has a doc in its header file, you don't need to add the doc to the source file.